pom.xml

```diff
@@ -151,9 +151,9 @@
 		<joda.time.version>2.12.7</joda.time.version>
 		<hibernate.validator.version>8.0.1.Final</hibernate.validator.version>
 		<jakarta.validation.api.version>3.1.0</jakarta.validation.api.version>
-		<opensearch.version>2.13.0</opensearch.version>
+		<opensearch.version>2.14.0</opensearch.version>
```

Comment:
Shouldn't this be 2.13.0?

Reply:
@sumerjabri for some reason, the bot updated it to 2.14. I reverted it to 2.13